From d44135a10325b5e4b8312709a39eae09da499538 Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vincent.rabaud@gmail.com>
Date: Tue, 10 May 2016 22:19:42 +0200
Subject: [PATCH] update the VTK and Qt compatibility Not sure why it's not
 picking up the package.xml from master. Probably because the package.xml used
 to be from upstream.

---
 package.xml | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/package.xml b/package.xml
index 1f9e1b9..1acd329 100644
--- a/package.xml
+++ b/package.xml
@@ -17,14 +17,12 @@
   <build_depend>jasper</build_depend>
   <build_depend>libjpeg</build_depend>
   <build_depend>libpng12-dev</build_depend>
-  <build_depend>libqt5-gui</build_depend>
   <build_depend>libtiff-dev</build_depend>
   <build_depend>libv4l-dev</build_depend>
-  <build_depend>libvtk</build_depend>
+  <build_depend>libvtk-qt</build_depend>
   <build_depend>protobuf-dev</build_depend>
   <build_depend>python</build_depend>
   <build_depend>python-numpy</build_depend>
-  <build_depend>qtbase5-dev</build_depend>
   <build_depend>zlib</build_depend>
 
   <buildtool_depend>cmake</buildtool_depend>
@@ -34,8 +32,7 @@
   <run_depend>jasper</run_depend>
   <run_depend>libjpeg</run_depend>
   <run_depend>libpng12-dev</run_depend>
-  <run_depend>libqt5-gui</run_depend>
-  <run_depend>libvtk</run_depend>
+  <run_depend>libvtk-qt</run_depend>
   <run_depend>protobuf</run_depend>
   <run_depend>python</run_depend>
   <run_depend>python-numpy</run_depend>
-- 
2.7.4

